### ISSUE TITLE
[exporter/debug] make the output stream configurable when using `use_internal_logger=false` 

### ISSUE THREAD

**Issue by jedr** [open-telemetry/opentelemetry-collector#10472]
**Is your feature request related to a problem? Please describe.**

With https://github.com/open-telemetry/opentelemetry-collector/pull/10227, users can set `use_internal_logger` to `false` to not use the collector's internal logger for output. In this case, the logs are sent to `stdout` and there is no way to change this.

**Describe the solution you'd like**

Add configuration option to the Debug exporter to change the output stream from `stdout` to e.g. `stderr` or maybe another file.

**Describe alternatives you've considered**

Leave this not configurable and have users redirect the standard output from the collector on operating system / console level.

**Additional context**

When `use_internal_logger` is `true` (currently the default), the exporter's output is sent to `stderr` together with all other logs from the collector. This configurable via `service::telemetry::logs::output_paths`.

**Comment by jackgopack4** [comment]
yeah this seems reasonable; it looks like there is a note in that section of debugexporter config that says "we should make this configurable" or something similar. I've opened a PR that should close this issue

**Comment by ishaish103** [comment]
@jackgopack4  @andrzej-stencel 
Hey ! 
i would like to work on this issue, may i be assigned?

**Comment by andrzej-stencel** [comment]
Sure @ishaish103, give it a go!

**Comment by Arunodoy18** [comment]
From my understanding, when use_internal_logger is set to false, the debug exporter bypasses the collector’s logging pipeline and writes directly to stdout, which makes the output stream non-configurable.

When use_internal_logger is true, the exporter output goes through the internal logger and therefore respects service.telemetry.logs.output_paths, which already allows routing to stderr or files. Disabling the internal logger currently removes that flexibility entirely.

This makes sense from an implementation standpoint, but it does create an inconsistency where users lose output control specifically in the `use_internal_logger: false path.

I would like to take on this .Hope this works. Cheers
